recipes/meteofetch/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "meteofetch" %}
+{% set version = "0.2.4" %}
+{% set python_min = "3.8" %}
```

Comment:
Conda-forge doesn't support EOL Python versions.  This variable is only when the package min is higher than our supported min, which is 3.9 now.

```suggestion
```